swssconfig/sample/00-copp.config.json

```diff
@@ -55,5 +55,21 @@
             "red_action":"drop"
         },
         "OP": "SET"
-    }
+    },
+    {
+        "COPP_TABLE:trap.group.sflow": {
+            "trap_ids": "sample_packet",
+            "trap_action":"trap",
+            "trap_priority":"1",
```

Comment:
This is taking the same priority as the ip2me?  I would think we should make it lower than that,  so maybe raise ip2me to "2", and keep this to "1".  to keep consistent with the queue, we can make ip2me to queue 2, and sflow to queue 1.